packages/styles/scss/patterns/sections/card-section/_card-section.scss

```diff
@@ -43,8 +44,36 @@
   }
 }
 
+@mixin ratio-base() {
```

Comment:
I wonder if we should pull this out as a utility until carbon get's their. mixins merged in.

Reply:
sure